config/src/main/java/org/springframework/security/config/annotation/web/configuration/WebSecurityConfiguration.java

```diff
@@ -88,7 +88,7 @@ public static DelegatingApplicationListener delegatingApplicationListener() {
 
 	/**
 	 * Creates the Spring Security Filter Chain
-	 * @return
+	 * @return {@link Filter} security filter chain.
```

Comment:
Maybe "the {@link Filter} that represents the security filter chain"? This phrasing is largely the same, but it has the benefit of being a complete clause.